src/compiler/coloring.nit

```diff
@@ -17,7 +17,7 @@ module coloring
 import poset
 
 # Build a conflict graph from a POSet
-class POSetConflictGraph[E: Object]
+class POSetConflictGraph[E]
```

Comment:
It seems to me that the `POSetConflictGraph` is quite specific and will complexify `lib/poset` without any real added value.